src/main/java/com/playtika/homework1/task2/DaysOfWeek.java

```diff
@@ -0,0 +1,37 @@
+package com.playtika.homework1.task2;
+
+import java.util.Scanner;
+
+public class DaysOfWeek {
+    public static void main(String[] args) {
+        System.out.print("Введите день недели: ");
+        Scanner scan = new Scanner(System.in);
+        String day = scan.next();
+
+        switch (day) {
```

Comment:
move switch statement to a separate method

Reply:
moved

Reply:
switched